src/main/scala/akka/contrib/http/Directives.scala

```diff
@@ -31,7 +32,7 @@ object Directives {
    */
```

Comment:
noticed the `accept` directive above which I'll want to discuss with the spray guys https://github.com/akka/akka/issues/19108
In one way it smells like an anti pattern (we have content negotiation built-in to handle these things), in another way though perhaps it could be useful. Will report back here